pkgs/development/python-modules/logassert/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  nix-update-script,
+  setuptools,
+  pytestCheckHook,
+  flake8,
+  structlog,
+}:
+
+buildPythonPackage rec {
+  pname = "logassert";
+  version = "8.2";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "facundobatista";
+    repo = "logassert";
+    tag = version;
+    hash = "sha256-wtSX1jAHanHCF58cSNluChWY3lrrsgludnnW+xVJuOo=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "logassert" ];
+
+  nativeCheckInputs = [
+    flake8
+    pytestCheckHook
+    structlog
+  ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
I think this is automatically added, maybe @gaetanlepage will confirm this?

Reply:
Fixed :)